.github/workflows/performance-check.yml

```diff
@@ -0,0 +1,118 @@
+name: Performance Check
+
+on:
+  pull_request:
+  merge_group:
+
+jobs:
+  build-plugin:
+    name: Build plugin
+    uses: ./.github/workflows/build.yml
+
+  compare-v4-to-v3:
+    name: Compare V4 to V3
+    runs-on: ubuntu-latest
+    needs: [build-plugin]
+    env:
+      PERFORMANCE_SCORE_THRESHOLD: 0.85
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v4
+
+      - name: Install Node.js 20.x
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+          cache: 'npm'
+
+      - name: Download build artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: ${{ needs.build-plugin.outputs.artifact_name }}
+          path: ./elementor
+
+      - name: Setup wp-env
+        uses: elementor/elementor-editor-github-actions/actions/setup-wp-env@main
+        with:
+          php: 7.4
+          active-theme: 'hello-elementor'
+          plugins: |-
+            ./elementor
+          themes: |-
+            https://downloads.wordpress.org/theme/hello-elementor.zip
+          mappings: |-
+            templates:./tests/lighthouse/compare-templates
+
+      - name: Setup Elementor ENV
+        uses: elementor/elementor-editor-github-actions/actions/setup-elementor-env@main
+        with:
+          env: 'testing'
+          enable-svg-upload: true
+          experiments: |-
+            e_optimized_markup:true
+            editor_v2:true
+            atomic_widgets:true
+            global_classes:true
+          templates: |-
+            templates
+
+      - name: Warm up the cache
+        run: |
+          curl -o /dev/null http://localhost:8889/v3-only/ &&
+          curl -o /dev/null http://localhost:8889/mixed-template/
+
+      - name: Run Performance Check
+        uses: elementor/elementor-editor-github-actions/actions/run-lighthouse-tests@main
+        id: run-lighthouse-tests
```

Comment:
shouldn't the id correspond to the name? 🫣

Reply:
Done